frontend/app/src/components/settings/accounting/rule/AccountingRuleWithLinkedSettingDisplay.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import { type AccountingRuleWithLinkedProperty } from '@/types/settings/accounting';
+import { useAccountingRuleMappings } from '@/composables/settings/accounting/rule-mapping';
+
+const props = defineProps<{
+  identifier: string;
+  item: AccountingRuleWithLinkedProperty;
+}>();
+
+const { identifier, item } = toRefs(props);
+
+const { t } = useI18n();
+
+const { accountingRuleLinkedMappingData } = useAccountingRuleMappings();
+
+const linkableSettingOptions = accountingRuleLinkedMappingData(identifier);
+
+const selectedLinkableSetting = computed(() => {
+  const itemVal = get(item);
+  const linkedProperty = itemVal.linkedSetting;
+  if (linkedProperty) {
+    const foundItem = get(linkableSettingOptions).find(
+      item => item.identifier === linkedProperty
+    );
+
+    if (foundItem) {
+      return foundItem;
+    }
+  }
+
+  return null;
+});
+
+const value: ComputedRef<boolean> = computed(() => {
+  const selectedLinkableSettingVal = get(selectedLinkableSetting);
+  if (selectedLinkableSettingVal) {
+    return get(get(selectedLinkableSettingVal).state);
```

Comment:
We should not need to do a get-get, the computed property should update any time any of the state updates if we unref inside right? does it not?